examples/research_projects/diffusion_orpo/train_diffusion_orpo_sdxl_lora.py

```diff
@@ -986,7 +986,9 @@ def compute_time_ids(original_size, crops_coords_top_left):
                 model_losses = F.mse_loss(model_pred.float(), target.float(), reduction="none")
                 model_losses = model_losses.mean(dim=list(range(1, len(model_losses.shape))))
                 model_losses_w, model_losses_l = model_losses.chunk(2)
-                log_odds = model_losses_w - model_losses_l
+                log_odds = (model_losses_w - model_losses_l) - (
+                    torch.log(1 - torch.exp(model_losses_w)) - torch.log(1 - torch.exp(model_losses_l))
```

Comment:
Did you validate this? Does it run well?

Reply:
checking do not merge

Reply:
I did check and it didn't train well. But let's see.